src/audio/module_adapter/module_adapter.c

```diff
@@ -844,6 +911,25 @@ int module_adapter_trigger(struct comp_dev *dev, int cmd)
 		return PPL_STATUS_PATH_STOP;
 	}
 
+	if (mod->num_input_buffers > 1) {
```

Comment:
doesn't this have to check for `.no_pause`?

Reply:
But I do check it just above when the trigger command is PAUSE. For the rest of the commands, only the state of the sources matter right?

Reply:
right, sorry, missed it